osu.Game.Tournament/Screens/Gameplay/GameplayScreen.cs

```diff
@@ -36,18 +41,41 @@ public class GameplayScreen : BeatmapInfoScreen, IProvideVideo
 
         private Drawable chroma;
 
+        private TourneyVideo fallbackGameplayVideo;
+        private readonly List<(string, TourneyVideo)> modSpecificGameplayVideos = new List<(string, TourneyVideo)>();
+
         [BackgroundDependencyLoader]
-        private void load(LadderInfo ladder, MatchIPCInfo ipc, Storage storage)
+        private void load(LadderInfo ladder, MatchIPCInfo ipc, TournamentStorage storage)
         {
             this.ipc = ipc;
 
+            var extraGameplayVideos = new List<TourneyVideo>();
+
+            // look for all video files that start with "gameplay-"
+            foreach (var name in storage.GetFiles("videos", "gameplay-*").Select(Path.GetFileNameWithoutExtension))
```

Comment:
No check on whether this exists or not is breaking tests.

Reply:
I added a check, hope this is ok.

I did earlier look into using `TournamentVideoResourceStore` (which is used in `TourneyVideo`) which sounds like it should give me a list of available videos directly without me having to check if the videos folder exists or not. I saw there is a `GetAvailableResources()` function available there but for whatever reason it always gives me an empty enumerable, so I resorted to using this. Not sure if I'm doing something wrong/misunderstanding something there or if theres a bug in the framework